doc/source/user_guide/metadata_structure.rst

```diff
@@ -0,0 +1,361 @@
+
+model
+    type: string
+
+    Model of the spectrometer (branding by manufacturer).
+
+acquisition_mode
+    type: string
+
+    Acquisition mode (e.g. 'Parallel dispersive', versus 'Serial dispersive').
+
+entrance_slit_width
+    type: float
+
+    Width of the entrance slit in mm.
+
+exit_slit_width
+    type: float
+
+    Width of the exit slit (serial acquisition) in mm.
+
+central_wavelength
+    type: float
+
+    Central wavelength during acquisition (parallel acquisition).
+    
+start_wavelength
+    type: float
+
+    Start wavelength in nm (serial acquisition).
+
+step_size
+    type: float
+
+    Step size in nm (serial acquisition).
+
+Grating
+-------
+
+Information of the dispersion grating employed in the measurement.
+
+groove_density
+    type: int
+
+    Density of lines on the grating in grooves/mm.
+
+blazing_angle
+    type: int
+
+    Angle in degree (º) that the grating is blazed at.
+
+blazing_wavelength
+    type: int
+
+    Wavelength that the grating blaze is optimized for in nm.
+
+Filter
+-------
+
+Information about additional filters entered into the lightpath after the
+sample. In case multiple filters are used, they should be numbered
+`Filter_1`, etc. See :ref:`filter-label` above for details on items that
+may potentially be included.
+
+Detector
+========
+
+Contains information about the detector used to acquire the signal. Contained
+leaves will differ depending on the type of detector.
+
+detector_type
+    type: string
+
+    The type of detector used to acquire the signal (CCD, PMT, StreakCamera, 
+    TCSPD).
+
+model
+    type: string
+
+    The model of the used detector.
+
+frames
+    type: int
+
+    Number of frames that are summed to yield the total integration time.
+
+integration_time (s)
+    type: float
+
+    Time over which the signal is integrated. In case multiple frames are
+    summed, it is the total exposure time. In case of serial acquisition, it is
+    the dwell time per data point.
+
+saturation_fraction
+    type: float
+
+    Fraction of the signal intensity compared with the saturation threshold of
+    the CCD.
+
+binning
+    type: tuple of int
+
+    A tuple that describes the binning of a parallel detector such a CCD on
+    readout in x and y directions.
+
+processing
+    type: string
+
+    Information about automatic processing performed on the data, e.g. 'dark
+    subtracted'.
+
+sensor_roi
+    type: tuple of int
+
+    Tuple that specifies range of pixels on a detector that are read out.
+
+pixel_width
+    type: float
+
+    Diameter of a pixel in µm.
+
+
+Spectral_image
+==============
+
+Contains information about mapping parameters, such as step size, drift
+correction, etc.
+
+mode
+    type: string
+
+    Mode of the spectrum image acquisition such as 'Map' or 'Linescan'.
+
+step_size
+    type: float
+
+    Distance between subsequent pixels in the spectral image.
+
+step_size_units
+    type: string
+
+    Units of the step size (standard 'nm').
```

Comment:
Is not redundant with the information provided in the `AxesManager`?

Reply:
Indeed, it is in the definition of the navigation axes, so we can leave it out here.